src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -51,20 +76,44 @@ const JsonLocalizer = () => {
             .map(([key, values]) => {
               const cellWidth = 100 / values.length;
               return (
-                <div className="table-row" key={key}>
-                  {/* <div className="cell" style={{ width: `${cellWidth}%` }}>
+                <>
+                  <div className='table-row' key={key}>
+                    {/* <div className="cell" style={{ width: `${cellWidth}%` }}>
                     {key}
                   </div> */}
-                  {values.map((v, i) => (
-                    <div
-                      key={i}
-                      className="cell"
-                      style={{ width: `${cellWidth}%` }}
-                    >
-                      {v}
+                    {values.map((v, i) => (
+                      <Cell
+                        key={`${key}-${i}`}
+                        id={`${key}-${i}`}
+                        style={{ width: `${cellWidth}%` }}
+                        value={v}
+                        onClick={(id) => cellOnClick(id)}
+                        shownInput={shownInput}
+                        className={
+                          shownInput === `${key}-${i}` ? 'active-cell' : ''
+                        }
+                      />
+                    ))}
+                  </div>
+                  {shownInput?.split('-')[0] === key && (
```

Comment:
Please, leave a comment about what parts does a `shownInput` consist of.

Reply:
It consists of 'row' and 'i'. In other words, this is id.  I will rename it to activeCellId